src/app/features/users-page/interfaces/user-api-response.ts

```diff
@@ -0,0 +1,61 @@
+export interface UserApiResponse {
+    results: UserApi[];
+}
+
+interface UserApi {
```

Comment:
name is confusing, is it API? no it is a user FROM API.
just name User in this case